### ISSUE TITLE
rename any TH_<service> string to TH_<service>_KEY

### ISSUE THREAD

**Issue by foxmask** [foxmask/django-th#188]
thus, this will follow [the requirements](https://docs.djangoproject.com/en/1.11/ref/settings/)

This changes should concern: 

* settings (forget this one, this will be handle in #189)
* code of each service in `th_<service>/my_<service>.py` module
* unit test in `th_<service>/tests.py` module

once it's done, running the UnitTest should check if anything is fine. (the test wont work until you do git pull from the contributor on issue #189, because you will rename but wont have the settings up2date)

**Comment by pohzipohzi** [comment]
Hi can I take this up?

**Comment by foxmask** [comment]
if you want to participate to the [hacktoberfest](https://hacktoberfest.digitalocean.com/), register and wait for october 1rst before submitting the PR.

**Comment by foxmask** [comment]
example with Evernote

from my_evernote.py 
```
class ServiceEvernote(ServicesMgr):
    """
        Service Evernote
    """
    def __init__(self, token=None, **kwargs):
        super(ServiceEvernote, self).__init__(token, **kwargs)
        self.sandbox = settings.TH_EVERNOTE['sandbox']
        self.consumer_key = settings.TH_EVERNOTE['consumer_key']
        self.consumer_secret = settings.TH_EVERNOTE['consumer_secret']
```
will become 
```
class ServiceEvernote(ServicesMgr):
    """
        Service Evernote
    """
    def __init__(self, token=None, **kwargs):
        super(ServiceEvernote, self).__init__(token, **kwargs)
        self.sandbox = settings.TH_EVERNOTE_KEY['sandbox']
        self.consumer_key = settings.TH_EVERNOTE_KEY['consumer_key']
        self.consumer_secret = settings.TH_EVERNOTE_KEY['consumer_secret']
```
from test.py
```
    def test_get_config_th(self):
        """
            does this settings exists ?
        """
        self.assertTrue(settings.TH_EVERNOTE)
```
will become

```
    def test_get_config_th(self):
        """
            does this settings exists ?
        """
        self.assertTrue(settings.TH_EVERNOTE_KEY)
```

**Comment by pohzipohzi** [comment]
I think we closed this in #193 as well!

**Comment by foxmask** [comment]
true ! 
Eye of the Tiger :)
